src/pages/index.astro

```diff
@@ -86,11 +86,12 @@ export const prerender = true;
   </div>
 
   <div class="bg-white text-red">
+  <div class="w-full flex justify-center my-12 text-4xl">✷</div>
```

Comment:
Added this star so that some content peeks out from under the hero and it's not just white